eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/body/client/HeartbeatResponseBody.java

```diff
@@ -27,13 +27,13 @@
 
 public class HeartbeatResponseBody extends Body {
 
-    //响应码
+    //react code
```

Comment:
return code may be more appropriate

Reply:
I think we may not need to add comments for each attribute, for example here, the `retCode` can express variable information. And I would probably prefer to rename it to `returnCode` rather than add a comment.